### PR TITLE
905652 - AI samples - Document Summarization Q&A

### PR DESCRIPTION
**Pull request by KSilambharasanSF4548**
**Task Details :**
**Name :** AI samples - Document Summarization Q&A
**Link** : https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_sprints/backlog/PdfViewerMAUI/Mobile%20and%20Desktop/2024Sep2_PDFViewer_MAUI?System.AssignedTo=%40me&workitem=905714

**Sample Description :**
An example added for how to leverage AI-driven document summarization and Q&A integrated with the .NET MAUI PDF Viewer, enabling users to quickly retrieve key insights and answers from lengthy documents for efficient content understanding.

**Demo :**

https://github.com/user-attachments/assets/a3088b3d-892d-48d7-ab34-307a68276666

### CODE REVIEW

File: AI Demos/Summarizer/Summarizer/AssistServices.cs [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+        /// <summary>
+        /// The EndPoint
+        /// </summary>
+        private string endpoint = "https://yourendpoint.com/";
+
+        /// <summary>
+        /// The Deployment name
+        /// </summary>
+        internal string deploymentName = "DEPLOYMENT_NAME";
```
Comment (VikramDevaraj31) [inline AI Demos/Summarizer/Summarizer/AssistServices.cs:28]: The casing should be proper for internal variable.

File: AI Demos/Summarizer/Summarizer/MainPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        }
+        else
+        {
+            StartBubbleAnimation();
+        }
+    }
+
+    internal async Task LoadPDFDataAsync()
+    {
+        await Task.Delay(1000);
```
Comment (VikramDevaraj31) [inline AI Demos/Summarizer/Summarizer/MainPage.xaml.cs:49]: Can reduce the ms and test.

File: AI Demos/Summarizer/Summarizer/AssistServices.cs [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        /// <summary>
+        /// The ChatCompletion option
+        /// </summary>
+        private ChatCompletionsOptions? chatCompletions;
+
+        internal AssistServices()
+        {
+            chatCompletions = new ChatCompletionsOptions
+            {
+                DeploymentName = DeploymentName,
```
Comment (VikramDevaraj31) [inline AI Demos/Summarizer/Summarizer/AssistServices.cs:49]: Mention "this.DeploymentName".

Comment (KSilambharasanSF4548) [reply AI Demos/Summarizer/Summarizer/AssistServices.cs]: Updated bro

File: AI Demos/Summarizer/Summarizer/MainPage.xaml.cs (hunk 2 above)
Comment (KSilambharasanSF4548) [reply AI Demos/Summarizer/Summarizer/MainPage.xaml.cs]: decreased to 100

File: AI Demos/Summarizer/Summarizer/AssistServices.cs (hunk 1 above)
Comment (KSilambharasanSF4548) [reply AI Demos/Summarizer/Summarizer/AssistServices.cs]: Updated bro

**Review by VikramDevaraj31 — APPROVED** [review]

**Pull request merged**